.github/pull_request_template.md

```diff
@@ -0,0 +1,58 @@
+<!-- Is there a relevant and understandable title succinctly summarizing the changes? -->
+
+# Description
+
+<!--- Give a description of the changes here in more detail -->
+<!--- Include screenshots if necessary -->
+
+## Related Issue
+
+<!--- This project only accepts pull requests related to already existing open issues -->
+<!--- Please link to the open issue here: -->
+
+Solves #<!--issue-->
+
+<!--Don't forget to move the related issue to the 'Review in progress' column.-->
+
+## Type of change
+
+<!--Please delete options that are not relevant.-->
+
+- [ ] New feature (enhancement)
+- [ ] Bug fix
+- [ ] Refactor
+- [ ] Breaking change
+- [ ] Documentation
+
+<!--Don't forget to add a label to the pull request, if relevant.-->
+
+## How Has This Been Tested?
+
+<!--- Please describe in detail how the changes have been tested. -->
+<!--- Include details of the testing environment, the tests run, and the results. -->
+
+<!--Please delete options that are not relevant.-->
+
+- [ ] Automated testing
+- [ ] Manual testing
+
+# Checklist:
+
+<!--Please delete options that are not relevant.-->
+
+- [ ] Does a similar (open or closed) pull request not already exist?
+- [ ] Is this pull request head a fork repository?
+- [ ] Is this pull request compare a development branch?
+- [ ] Is the code documented, particularly in hard-to-understand areas?
+- [ ] Does the code build without new warnings?
+- [ ] Has testing been performed that proves changes are effective and work?
+- [ ] Has a self- and/or peer-review of the code been performed?
+- [ ] Have dependent changes been merged and published in downstream modules?
+- [ ] Does all new and existing automated testing pass?
+- [ ] Is the person responsible for the repository assigned to the pull request?
+- [ ] Is the pull request linked to a project?
+- [ ] Is the pull request linked to a milestone?
+
```

Comment:
# Screenshot
<!-- if necessary, add the screenshot to show what the previous and current one looks like -->
Original                   |Updated
:---------------------:|:------------------:
**original screenshot ** | ** updated screenshot **|

Reply:
Thank you for your review and feedback @Xiaoqi510 

Regarding using before and after screenshots for reviewing code changes, there is already the suggested option for adding screenshots in the template description and I don't think it is necessary to have screenshots as a requirement when we can already view code changes in the pull request commit(s).

Reply:
You are right! I forget the "files-changed" function of github :)